graph/pull.go

```diff
@@ -555,6 +567,12 @@ func (s *TagStore) pullV2Tag(r *registry.Session, out io.Writer, endpoint *regis
 			// if tmpFile is empty assume download and extracted elsewhere
 			defer os.Remove(d.tmpFile.Name())
 			defer d.tmpFile.Close()
+
+			if !d.validChecksum {
+				out.Write(sf.FormatProgress(stringid.TruncateID(d.img.ID), "Checksum problem", nil))
```

Comment:
instead of `Checksum problem` this should be the string value of `errImageChecksumFailed`, but also maybe this should be written by the caller since we are returning an error.